pkg/snapshot/restorer/restorer_test.go

```diff
@@ -436,6 +468,82 @@ var _ = Describe("Running Restorer", func() {
 				// behavior of etcd restart.
 			})
 		})
+
+		Context("when snapshots are compressed using different compressionPolicy", func() {
```

Comment:
I like the concept of adding snapshots with different compression algorithms into the same snapstore and then restoring successfully from that and testing that this works. Nice! 🙂